packages/jitar-reflection/test/_fixtures/parser/Lexer.fixture.ts

```diff
@@ -1,13 +1,18 @@
 
 const CODE =
 {
-    OPERATORS: `=====!=/=/!=`,
+    OPERATORS: `=====!=+=*!=`,
```

Comment:
Because the / now conflicts with the regex determination. In real code this operator string can not exist without having errors, but I've created this to make sure the separation of operators works. With the replacements we still test the same situation; does it separate an arithmetic assignment from the next arithmetic operation. We've had an issue with this earlier.